tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist =
+    py27-django{111},
+    py37-django{111,20},
+    flake8
+
+[testenv]
+deps =
+    pytest
+    case
```

Comment:
you need to specify how the `django` factor influences the environment. I.e. by adding the following lines here:

```
    django111: Django>=1.11,<1.12
    django20: Django>=2.0,<2.1
    django21: Django>=2.1,<2.2
```